### PR TITLE
Add pipenv requirements

### PR DESCRIPTION
**Pull request by asyd**
Once pipenv is installed, simply do:

pipenv install
pipenv run python gricleaner.py

For information, pipenv is the new kind of official way to manage dependencies.

### CODE REVIEW

**Comment by n0madic** [comment]
The utility also works with Python 3, how can I specify this?

**Comment by asyd** [comment]
`pipenv --three install` should work

File: Pipfile [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+python-dateutil = "*"
+
+
+[dev-packages]
+
+
+
+[requires]
+
+python_version = "2.7"
```
Comment (n0madic) [inline Pipfile:21]: I'm confused by this line

Comment (asyd) [reply Pipfile:21]: Feel free to remove this section, according to https://docs.pipenv.org/basics/#specifying-versions-of-python

**Pull request merged**